ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'app-faq-section',
+    templateUrl: './faq-section.component.html',
+    styleUrls: ['./faq-section.component.scss']
+})
+export class FaqSectionComponent {
+    @Input() questions: string;
+    @Input() questionsCount: number;
```

Comment:
#minor we can add a test which verifies that questionsCount works correctly (count number of question elements using queryAll)

Reply:
Added at d7832afe1fba109ce5580e25a1afaa8c9d6cd4b7